tutorials/themes/tutorials-theme/layout.html

```diff
@@ -27,7 +27,7 @@
 <!DOCTYPE html>
 {%- endblock %}
 
```

Comment:
Is this a stylistic change? And are we sure both names are valid?

Reply:
I changed `navBar` to `nav_bar` so that there will definitely be no confusion in case the system used is not case sensitive.